utils/releasetools/build-config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "linux_targets": [
+        {
+          "arch": "x86_64",
+          "target": "ubuntu18.04",
+          "type": "deb",
+          "platform": "bionic"
+        },
+        {
+          "arch": "x86_64",
+          "target": "ubuntu22.04",
+          "type": "deb",
+          "platform": "focal"
+        },
+        {
+          "arch": "arm64",
+          "target": "ubuntu18.04",
+          "type": "deb",
+          "platform": "bionic"
+        },
+        {
+          "arch": "arm64",
+          "target": "ubuntu20.04",
+          "type": "deb",
+          "platform": "focal"
+        }
+      ],
+    "macos_targets": [
+        {
+          "arch": "x86_64",
+          "target": "catalina",
+          "type": "zip",
+          "platform": "catalina"
+        }
+      ]
```

Comment:
Latest macs are on ARM and would prefer us avoiding the rossetta layer. Can we also support mac arm arch type?

Reply:
Yes se can support it, but I am waiting on the the info for signing the mac binaries

Reply:
Let's file an issue for this.

Reply:
Yeah, we won't have that today for sure.